### ISSUE TITLE
[Eval Tools list] Contrast of the focus indicator in the "Start filter assistant" and "Submit a tool" buttons is insufficient

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-eval-tools#98]

**Comment by VeraLange** [comment]
We changed the boxes around these buttons to help the focus indicator stand out more. However, the buttons themselves are a standard component we have little control over. 

Possibly another issue to be discussed on the topic of WAI components, @shawna-slh  ?
